prompb/remote.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package remote;
```

Comment:
Mh, should we really have two proto packages in the same dir.  Maybe it's easiest to just have a `prometheus` package – we can break them up when we find them getting too busy.
The admin APIs are also in the `prometheus` package.